_static/basic.css

```diff
@@ -4,7 +4,7 @@
  *
```

Comment:
did the styling get updated by memo or should that be a different PR?

Reply:
Hi @munkm and @yardasol . I saw your comments on this PR. it seems you are having an issue to figure out the differences from the previous PR. So, let me explain. What you are looking at is gh-pages containing html and supporting documents for https://arfc.github.io/saltproc/. I updated these files in a separate PR. You need to create all these files all over again when you change any file under saltproc/doc/ by using Sphinx. So, you may not be able to see the difference by directly looking into files in gh-pages. For your convenience, I put below the update I made in that PR,
In "overview.html", 

Updates
March 2021:

Besides the existing flexibility like fixed removal efficiency definition for each target isotope defined in the object input file, Saltproc code now comprises Sparging System package that calculates removal efficiencies for various target isotopes (i.e., Xe, Kr, and H). To enable this feature, use the “self” command in the input file in the “efficiency” object names of Sparger and Separator components. Each component can be employed separately. An example is given below. In the example, capacity and mass flow rate are in g/s unit while volume is in cm3 unit.

“sparger”: { “capacity”: 9920000,
“efficiency”: “self”, “mass_flowrate”: 9920000, “volume”: 10000000 },

“entrainment_separator”: { “capacity”: 9920000,
“efficiency”: “self”, “mass_flowrate”: 9920000, “volume”: 11 }

Please for all changes I made, take a look at my last PR in master, not in gh-pages.